code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -265,7 +265,7 @@ GLOBAL_LIST_INIT(nonreactive_gases, typecacheof(list(/datum/gas/oxygen, /datum/g
 	appearance_flags = TILE_BOUND
 	vis_flags = NONE
 
-/obj/effect/overlay/gas/New(state, alph)
+/obj/effect/overlay/gas/Initialize(mapload, state, alph)
```

Comment:
This also makes me just a mite nervous, but I'm pretty sure any issues would fall out pretty fast so it should be fine